README.md

```diff
@@ -1,156 +1,74 @@
-| `@kadena/marmalade-client`       | Specific client for marmalade/NFTs                                                                                                                                                                                                                                                                                                               |
-| `@kadena/types`                  | Common used TypeScript definitions. Preferably this will all move to the low-level libraries. Whenever you build something on top of those, you can use those types.                                                                                                                                                                             |
-| `@kadena/pactjs`                 | Runtime for generating pact expressions                                                                                                                                                                                                                                                                                                          |
-| `@kadena/pactjs-cli`             | CLI to generate pact contract type definitions and interface to pact client. deployment of contracts, etc                                                                                                                                                                                                                                        |
-| `@kadena/pactjs-generator`       | Library that creates TypeScript definition from contract, template, etc                                                                                                                                                                                                                                                                          |
-| `.kadena/pactjs-generated`       | Library that does not exist on npm. It is *generated* by `@kadena/pactjs-cli` and `@kadena/pactjs-generator` and is a dependency of `@kadena/pactjs`                                                                                                                                                                                             |
-| `@kadena/transaction-templates`  | Supportive library for transactions. As there is no way to determine from pact alone which caps are needed for a given transaction, and in turn which signatures are needed, we want to provide the community a way to "publish" templates. These templates can be used by `@kadena/pactjs-cli` to generate the necessary TypeScript definitions |
-
-As our ecosystem will grow so will the packages and dapps we will release under
```

Comment:
Will we have this somewhere else?

Reply:
This is why I asked:

> Do we agree on the list of "main packages" in README.md?

Happy to hear your thoughts, I think we should have a single table. From the table we can link to the packages where folks will find the readme/description etc.